packages/flutter_android_lifecycle/lib/flutter_android_lifecycle.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+class FlutterAndroidLifecycle {
+  static const MethodChannel _channel =
+      const MethodChannel('flutter_android_lifecycle');
+
+  static Future<String> get platformVersion async {
```

Comment:
I'd definitely remove this, I'm curious if we can publish a pub package with no Dart code (worth trying). If not we should figure out what's the minimal contents we need here (an empty file?)